### ISSUE TITLE
Median filter error: Failed to execute 'importScripts' on 'WorkerGlobalScope': The script at 'http://localhost:9999/itk/Pipelines/itkfilteringWasm.js' failed to load.

### ISSUE THREAD

**Issue by XuJiachengZust** [Kitware/glance#470]
ParaViewGlance: master
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36 Edg/111.0.1661.54

```
Median filter error: Failed to execute 'importScripts' on 'WorkerGlobalScope': The script at 'http://localhost:9999/itk/Pipelines/itkfilteringWasm.js' failed to load.
```

**Comment by floryst** [comment]
I'm guessing this is due to build artifacts being incorrectly referenced. I can look into this.

**Comment by Gemost** [comment]
After I cloned the project to local, I modified the config code, 
        {
          from:path.join(paths.itk_path,'web-build'),
          to:path.join('itk', 'Pipelines'),
          // to: '[name][ext]',
        },
which did not report error failed to load, but then reported error **Median filter error: undefined**

**Comment by Gemost** [comment]
> 



> I'm guessing this is due to build artifacts being incorrectly referenced. I can look into this.

After I cloned the project to local, I modified the config code, 
        {
          from:path.join(paths.itk_path,'web-build'),
          to:path.join('itk', 'Pipelines'),
          // to: '[name][ext]',
        },
which did not report error failed to load, but then reported error **Median filter error: undefined**
How do I fix it

**Comment by floryst** [comment]
This should be fixed with #478 and #479.
